src/v2/layouts/MemberDataGrid/MemberInfoButtons/MobileButtons.jsx

```diff
@@ -1,47 +1,49 @@
+import { Icon } from '../../../components/design/Icon/Icon';
 import { faArrowLeft } from '@fortawesome/free-solid-svg-icons/faArrowLeft';
 import { faUsers } from '@fortawesome/free-solid-svg-icons/faUsers';
 import { faFile } from '@fortawesome/free-solid-svg-icons/faFile';
 import { faCog } from '@fortawesome/free-solid-svg-icons/faCog';
 import { faPowerOff } from '@fortawesome/free-solid-svg-icons/faPowerOff';
 import { faSync } from '@fortawesome/free-solid-svg-icons/faSync';
-import { Icon } from '../../components/design/Icon/Icon';
+import { PageRoutes } from '../../../components/providers/RouteProvider/RouteProvider';
 import styled, { css } from 'styled-components';
 import { Link } from 'react-router-dom';
+import PropTypes from 'prop-types';
 
-export type MemberInfoButtonsProps = {
-	setIsVisibleConfigModal: (visibleConfigModal: boolean) => void;
-};
-
-export const MemberInfoButtons = (props: MemberInfoButtonsProps): JSX.Element => {
+export const MobileButtons = ({ setIsVisibleConfigModal, refreshStudentsDataGrid }) => {
+	const { MEMBER_REGISTRATION, LOGIN } = PageRoutes;
 	return (
-		<MobileButtonsContainerStyled>
-			<MobileButtonsBoxStyled>
-				<LinkStyled to={''}>
-					<Icon icon={faArrowLeft} />
-				</LinkStyled>
-				<LinkStyled to={''}>
-					<Icon icon={faUsers} />
-				</LinkStyled>
-				<StyledFileLink to={''}>
-					<Icon icon={faFile} />
-				</StyledFileLink>
-				<StyledButton
-					onClick={() => {
-						props.setIsVisibleConfigModal(true);
-					}}>
-					<Icon icon={faCog} />
-				</StyledButton>
-				<LinkStyled to={''}>
-					<Icon icon={faPowerOff} />
-				</LinkStyled>
-				<StyledButton>
-					<Icon icon={faSync} />
-				</StyledButton>
-			</MobileButtonsBoxStyled>
-		</MobileButtonsContainerStyled>
+		<MobileButtonsBoxStyled>
+			<LinkStyled to={LOGIN}>
+				<Icon icon={faArrowLeft} />
+			</LinkStyled>
+			<LinkStyled to={MEMBER_REGISTRATION}>
+				<Icon icon={faUsers} />
+			</LinkStyled>
+			<StyledFileLink to={''}>
+				<Icon icon={faFile} />
+			</StyledFileLink>
+			<StyledButton
+				onClick={() => {
```

Comment:
I think we should use it as a callback instead
```suggestion
				onClick={onClickConfigBtn}
```